slam_toolbox/lib/karto_sdk/include/karto_sdk/Mapper.h

```diff
@@ -927,6 +933,11 @@ namespace karto
       ar & BOOST_SERIALIZATION_NVP(m_pTraversal);
     }
 
+    /**
+     * Member bool variable to keep track of the state of a loop closure.
+     */
+    kt_bool m_pLoopClosed;
```

Comment:
This member variable should be serialized. See below in this class, its an easy 1-liner.

Reply:
Sure but to be clear, you mean renaming `m_pLoopClosed` to `optimizer_cached`?

Reply:
well still `m_pOptimizerCached` to follow convention